### PR TITLE
fixed typo: rcm -> rvm

### PR DESCRIPTION
**Pull request by cashmann**
fixed typo: rcm -> rvm

### CODE REVIEW

File: mac [hunk 1]
```diff
@@ -117,7 +117,7 @@ tap "caskroom/cask"
 # Unix
 brew "git"
 brew "openssl"
-brew "rcm"
+brew "rvm"
```
Comment (ghost) [inline mac]: Now that I took another look, I think we can remove this.  I'm 99.9% sure we've all switched to `rbenv`.

Comment (noahc) [reply mac]: Or replace it with rbenv if it doesn't already exist.

Comment (cashmann) [reply mac]: rbenv is in the file. I haven't had to use rvm at all, so I went ahead and deleted it.

**Comment by noahc** [comment]
+1

**Pull request merged**